### PR TITLE
DE44939 Fix for Console Warning and Error

### PR DESCRIPTION
**Pull request by Syeda-Azim**
https://rally1.rallydev.com/#/431372673576d/iterationstatus?detail=%2Fdefect%2F607593449267

### CODE REVIEW

File: src/components/catoc-course-selection.js [hunk 1]
```diff
@@ -123,6 +123,7 @@ class CatocCourseSelection extends LocalizeMixin(LitElement) {
 		return html`
 			<d2l-list-item
 				key=${course.OrgUnit.Id}
+				label=${course.OrgUnit.Id}
```
Comment (IhabAlkabary) [inline src/components/catoc-course-selection.js:126]: why same info repeated, why not the OrgUnit Name

File: src/components/catoc-summary.js [hunk 2]
```diff
@@ -78,16 +79,23 @@ class CopyAssignmentToOtherCoursesSummary extends LocalizeMixin(LitElement) {
 	}
 
 	render() {
-		const rubrics = this._dropbox.Assessment.Rubrics;
+		let rubrics = [];
+
+		if (!this.isQuerying) {
+			rubrics = this._dropbox.Assessment.Rubrics;
```
Comment (jorge-ramirez-arredondo) [inline src/components/catoc-summary.js:85]: This is a really minor nitpick, so don't worry about it, but if you used a ternary here, `rubrics` can stay a `const` variable.

**Comment by lweber-d2l** [comment]
In the rally case, one of the console warnings is "d2l-tooltip may be being used in a non-accessible manner; it should be attached to interactive elements..." - is this warning addressed here?  I don't see a change related to `d2l-tooltip`.

File: src/components/catoc-summary.js [hunk 3]
```diff
@@ -78,16 +79,23 @@ class CopyAssignmentToOtherCoursesSummary extends LocalizeMixin(LitElement) {
...
+		}
 
 		return html`
 			<div class="d2l-summary" >
 				<h4>${this._dropbox.Name}</h4>
 				<p class="title">${this._course.Name} (${this._course.Code})</p>
 				<div class="d2l-checkbox-container">
-					<d2l-input-checkbox id="rubric-checkbox">
+					<input
```
Comment (Syeda-Azim) [inline src/components/catoc-summary.js:93]: Warning for tooltip is addressed here.

Currently tooltips can’t cross shadow DOM boundaries — so we want the tooltip to be tied to the <input type="checkbox" but that’s buried inside d2l-input-checkbox’s shadow DOM.

Workaround is to use a native <input type="checkbox"> and just style it like our checkboxes, and then point the tooltip at that.

**Comment by Syeda-Azim** [comment]
> In the rally case, one of the console warnings is "d2l-tooltip may be being used in a non-accessible manner; it should be attached to interactive elements..." - is this warning addressed here? I don't see a change related to `d2l-tooltip`.

https://github.com/Brightspace/copy-assignment-to-other-courses/pull/37#discussion_r699484730

File: src/components/catoc-summary.js (hunk 3 above)
Comment (lweber-d2l) [reply src/components/catoc-summary.js:93]: Cool, thanks for the explanation!

**Review by lweber-d2l — APPROVED** [review]

**Pull request merged**